src/object/cli_obj.c

```diff
@@ -4671,22 +4671,23 @@ sgls_set_merged_bitmap(struct sgl_merge_ctx *ctx, d_sg_list_t *sg, uint32_t frag
  * Return: 0 on success, negative error code on failure
  */
 static int
-obj_sgls_dup(struct obj_auxi_args *obj_auxi, daos_obj_update_t *args, bool update)
+obj_processed_sgls(struct obj_auxi_args *obj_auxi, daos_obj_update_t *args, bool update)
```

Comment:
the naming, maybe obj_process_sgls() or obj_sgls_process() is better than the "ed" version

Reply:
OK